dpdk.spec

```diff
@@ -297,28 +269,33 @@ sed -i -e 's:-%{machine_tmpl}-:-%{machine}-:g' %{buildroot}/%{_sysconfdir}/profi
 %{incdir}/
 %{sdkdir}/
 %if %{with tools}
-%exclude %{sdkdir}/usertools/
 %endif
 %if %{with examples}
 %exclude %{sdkdir}/examples/
 %endif
 %{_sysconfdir}/profile.d/dpdk-sdk-*.*
 %{_libdir}/*.so
+%{pmddir}/*.so
+%{_libdir}/*.a
 %if %{with examples}
 %files examples
 %exclude %{_bindir}/dpdk-procinfo
 %exclude %{_bindir}/dpdk-pdump
 %{_bindir}/dpdk-*
 %doc %{sdkdir}/examples/
 %endif
+%{_libdir}/pkgconfig/*.pc
 
 %if %{with tools}
 %files tools
-%{sdkdir}/usertools/
-%{_sbindir}/dpdk-devbind
+%{_bindir}/dpdk-*.py
 %endif
 
 %changelog
+* Fri Jun 21 2021 Tom Nabarro <tom.nabarro@intel.com> - 0:21.02-1
```

Comment:
It seems a bit of time travel here.  :-)  When you get back, let me know what the winning lottery numbers were for that day.  :-D

That killed your PR right at linting unfortunately.